src/core/agent/services/identifierService.test.ts

```diff
@@ -354,13 +353,12 @@ describe("Single sig service of agent", () => {
     ).rejects.toThrowError(IdentifierService.THEME_WAS_NOT_VALID);
   });
 
-  // For archive/delete/restore tests
+  // For delete/restore tests
```

Comment:
We can remove this comment. No such thing as restore anymore, and not really needed